Form-Controls/styles.css

```diff
@@ -0,0 +1,151 @@
+/* Basic Reset */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* Body styling */
+body {
+    font-family: Arial, sans-serif;
+    line-height: 1.6;
+    background-color: #FBFDFC; /* Light background */
+    color: #333; /* Dark text for readability */
+}
+
+/* Container */
+.container {
+    width: 80%;
+    margin: 0 auto;
+    overflow: hidden;
+}
+
+/* Header styling */
+header {
+    background: #c4c4c4; /* Darker gray for better visibility */
+    color: #1a1a1a; /* Darker text for contrast */
+    padding: 20px 0;
+    text-align: center;
+}
+
+/* Main content */
+main {
+    padding: 20px;
+    background: #ffffff; /* White for main content */
+    margin-top: 20px;
+    border-radius: 5px;
+}
+
+form {
+    display: flex;
+}
+
+.field-wrap {
+    margin-bottom: 40px;
+}
+
+.formbox {
+    display: flex;
+    flex-direction: column;
+    gap: 20px; /* Better than justify-content for consistent spacing */
+    padding: 20px; /* Adds spacing inside the container */
+    border: 1px solid #ccc; /* Optional: adds a border for a defined area */
+    border-radius: 5px; /* Rounded corners for a more modern look */
+    background-color: #f0f0f0; /* Optional: background color for better visibility */
+    width: fit-content;
+}
+
+input, select, button {
+    color: #1a1a1a; /* Darker text for better contrast */
+    background-color: #ffffff; /* White background for form controls */
```

Comment:
I notice that your form control backgrounds are not white -- what is your intention?

Reply:
I'm not haveing really good realtions with colors but for take 100% accebility i should do it.